tests/copilot_helper/utils/fixtures/addons_files/s3_addons_policy.yml

```diff
@@ -0,0 +1,13 @@
+##### s3-policy example:
+
+my-s3-policy:
+  type: s3-policy  # gives the listed services permissions to access an existing bucket
+  readonly: true
+  deletion-policy: Retain
+  services:  # services that require access to the bucket.
+    - "web"
+    - - "other"
```

Comment:
```suggestion
    - "other"
```

Reply:
That's interesting. I'd have expected that to fail the test since it isn't a string.